File: app/src/main/java/com/abona_erp/driver/app/data/model/PercentItem.java
package com.abona_erp.driver.app.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PercentItem {
  
  @SerializedName("PercentFinished")
  @Expose
  private Double percentFinished;
  
  @SerializedName("TotalPercentFinished")
  @Expose
  private Double totalPercentFinished;
  
  public Double getPercentFinished() {
    return percentFinished;
  }
  
  public void setPercentFinished(Double percentFinished) {
    this.percentFinished = percentFinished;
  }
  
  public Double getTotalPercentFinished() {
    return totalPercentFinished;
  }
  
  public void setTotalPercentFinished(Double totalPercentFinished) {
    this.totalPercentFinished = totalPercentFinished;
  }
}

File: app/src/main/java/com/abona_erp/driverapp/data/model/PercentItem.kt
package com.abona_erp.driverapp.data.model


import com.google.gson.annotations.SerializedName

data class PercentItem(
    @SerializedName("MandantId")
    val mandantId: Int,
    @SerializedName("OrderNo")
    val orderNo: Int,
    @SerializedName("PercentFinished")
    val percentFinished: Double,
    @SerializedName("TaskId")
    val taskId: Int,
    @SerializedName("TotalPercentFinished")
    val totalPercentFinished: Double
)